tests/fsharp/core/nameof/version46/test.fsx

```diff
@@ -341,68 +341,3 @@ module Foo =
     let nameof = ()
     let x = name ()
 
-do test "local variable name lookup"                    (BasicNameOfTests.``local variable name lookup`` ())
```

Comment:
Why were these removed?

Reply:
The test is for generating errors, not for execution, I find it confusing if they're there but not actually being executed